securebanking-openbanking-uk-rcs-sample/src/test/resources/application-test.yml

```diff
@@ -1,13 +1,21 @@
-dns:
-  hosts:
-    root: hosts-root
-
 rcs:
   issuerId: forgerock-rcs
 
-rs-store:
+rs:
   internal-port: 8443
-  base-url: https://rs-store:${rs-store.internal-port}
+  base-url: https://ig:${rs.internal-port}
+
+idm:
+  internal-port: 8444
+  base-url: https://ig:${idm.internal-port}
+
+jwkms:
+  internal-port: 8555
+  base-url: https://ig:${jwkms.internal-port}
+
+dns:
+  hosts:
+    root: hosts-root
```

Comment:
There's quite a bit to sort out here in terms of how it's all going to fit together. Not sure if we need this `dns.hosts` property for example.

I like the idea of having a simple `base-url` for each service and a consistent way to configure each one. I've done this here for the rs, idm and jwkms, but I've copied some of the am config from the OB toolkit and it's quite a bit more complicated.